.claude/skills/deep-review/SKILL.md

```
@@ -15,6 +15,22 @@ Current branch: !`git branch --show-current`
 Recent commits on this branch:
 !`git log --oneline -20`
 
+## Pre-flight: Gemini availability
+
+Before launching any agents, verify Gemini is available by running this smoke test using the Bash tool:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)